### ISSUE TITLE
WebView2 can now be mentioned as an alternative to Edge canary in MBB experimental docs

### ISSUE THREAD

**Issue by TheCakeMonster** [dotnet/MobileBlazorBindings#337]
As a result of a conversation on one of Steve Sanderson's repos, I can confirm that the documentation on MBB is now slightly out of date. The WebView2 runtime is now available on a number of platforms, and this seems to work just as well as Microsoft Edge canary. This means that installation of the WebView2 runtime is probably a valid alternative to use of a canary build of Edge.

Can I suggest another update to the docs at https://docs.microsoft.com/en-us/mobile-blazor-bindings/get-started to indicate that either Microsoft Edge canary or installation of the WebView2 runtime are the prerequisites for the experimenting with the MBB preview?

I'd have done this myself, except that this page doesn't appear to be editable using GitHub pull requests, which is available on a number of other .NET documentation pages.

Many thanks.

**Comment by Eilon** [comment]
Yup, I will update this. The pages are editable via https://github.com/MicrosoftDocs/mobile-blazor-bindings but I already plan to do this update.
